runner.go

```diff
@@ -822,11 +822,16 @@ func atomicWrite(path string, contents []byte, perms os.FileMode, backup bool) e
 		}
 	}
 
-	f, err := ioutil.TempFile(parent, "")
+	d, err := ioutil.TempDir(os.TempDir(), "")
```

Comment:
I agree, I almost did that -- but I grepped the codebase and found that [os.TempDir is used with the instances of ioutil.TempFile](https://github.com/hashicorp/consul-template/search?utf8=✓&q=Tempdir).  Easy change either way.